trailblazer/store/models.py

```diff
@@ -92,6 +92,7 @@ class Analysis(Model):
     version = Column(types.String(32))
     workflow = Column(types.String(32))
     workflow_manager = Column(types.Enum(*WorkflowManager.list()), default=WorkflowManager.SLURM)
+    tower_workflow_id = Column(types.String(32))
```

Comment:
Should we default to `None` here, or do you plan to set it for the SLURM-analyses as well?

Reply:
https://github.com/Clinical-Genomics/trailblazer/pull/483/commits/d482bc7a2a8de5030714d5a5ed71995d51a77e94